portal/models/message.py

```diff
@@ -180,7 +180,8 @@ def send_message(self, cc_address=None):
         else:
             # This should never happen, alert if it does
             current_app.logger.error(
-                "Unable to generate audit log for email {}".format(Message))
+                "Unable to generate audit log for email {}".format(
+                    str(Message)))
```

Comment:
I think loggers use printf-type formatting by default. maybe the reason it's not casting `Message`?
eg:
```
current_app.logger.error("Unable to generate audit log for email %s", Message)

```